### PR TITLE
Add requirements.txt

### PR DESCRIPTION
**Pull request by StykMartin**
Signed-off-by: Martin Styk <mastyk@redhat.com>

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ansible
```
Comment (duck-rh) [inline requirements.txt]: I would require at least 2.7 here, as I like to use new powerful features ;-P

**Review by duck-rh — CHANGES_REQUESTED** [review]
just a minor remark

File: requirements.txt (hunk 1 above)
Comment (StykMartin) [reply requirements.txt]: Sounds good. Added :rocket: .

**Review by duck-rh — APPROVED** [review]
LGTM

**Pull request merged**